bundles/org.openhab.binding.intesis/README.md

```diff
@@ -0,0 +1,83 @@
+# Intesis Binding
+
+This binding connects to WiFi [IntesisHome](http://www.intesishome.com/) devices using their local REST Api.
+It does actually not support [IntesisBox](http://www.intesisbox.com/) devices but support is planned in upcoming version.
+
+
+
+## Supported Things
+
+This binding only supports one thing type:
+
+| Thing      | Thing Type | Description                                                            |
+|------------|------------|------------------------------------------------------------------------|
+| intesisHome | Thing      | Represents a single WiFi device                                         |
+
+## Discovery
+
+Intesis devices do not support auto discovery.
+
+## Thing Configuration
+
+The binding needs two configuration parameters, passwort and IP-Address.
+
+## Channels
+
+| Channel ID | Item Type          | Description                                                           | Possible Values |
```

Comment:
```suggestion
| Channel ID | Channel Type          | Description                                                           | Possible Values |
```

Reply:
No, this describes that the channel has to be linked to a certain item Type. This is common usage in the readme files.